core/src/test/resources/newlineBeforeLambdaParams/LambdaNewline.stat

```diff
@@ -0,0 +1,12 @@
+40 columns                              |
+<<< newline before lambda params {
+lst.map { x =>
+      println(x)
+  x + 1
+}
+>>>
+lst.map {
+  x =>
+    println(x)
+    x + 1
+}
```

Comment:
In this case in particular I'd like one-liners to be preserved, unless explicitly indented... but I'm struggling with it.